package.json

```diff
@@ -77,6 +77,10 @@
   "c8": {
```

Comment:
Should we move the c8 config to its own file? I notice you prefer keeping package.json clean. :)

Reply:
I do like a clean package.json :P. Either way is honestly fine with me.  I wouldn't say no to a .c8rc!